src/bundler.ts

```diff
@@ -386,12 +387,93 @@ class Bundler {
    * Given a URL to an entry-point html document, produce a single document
    * with HTML imports, external stylesheets and external scripts inlined,
    * according to the options for this Bundler.
+   *
+   * Given Multiple urls, produces a sharded build.
    */
-  async bundle(url: string): Promise<DocumentCollection> {
+  async bundle(bundles: string[], heuristic?: BundleOrchestrator):
+      Promise<DocumentCollection> {
     if (!this.analyzer) {
       throw new Error('No analyzer provided.');
     }
-    const analyzedRoot = await this.analyzer.analyzeRoot(url);
+    const doc = await this._bundleDocument(
+        bundles[0], this.excludes, this.stripExcludes, this.analyzer);
+    const collection = new Map<string, ASTNode>();
+    collection.set(bundles[0], doc);
+    return collection;
+    // const depsIndex = buildDepsIndex(bundles, this.analyzer) const manifest =
+    //     this._bundleManifest() return this._bundleMultiple(url,
+    //     this.analyzer);
+  }
+
+
+
+  private async _bundleMultiple(entrypoints: string[], analyzer: Analyzer):
```

Comment:
Looks like we keep passing an analyzer around. Could we just add it as a constructor argument?